### PR TITLE
Feat/ Toast

### PR DESCRIPTION
**Pull request by SalonSikdar**
Review comments & Toast component

### CODE REVIEW

File: css/comp-page.css [hunk 1]
```diff
@@ -129,8 +129,9 @@
   line-height: 1.5em;
   color: #464d5f;
```
Comment (05him) [inline css/comp-page.css]: i thnk you forget to use color variable here

File: css/main.css [hunk 2]
```diff
@@ -117,3 +118,41 @@ button:hover {
...
+
+.btn-close {
+  box-sizing: content-box;
+  width: 1em;
+  height: 1em;
+  padding: 0.25em 0.25em;
+  color: var(--black-global);
+  background: transparent
+    url("data:image/svg+xml,%3csvg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 16 16' fill='%23000'%3e%3cpath d='M.293.293a1 1 0 011.414 0L8 6.586 14.293.293a1 1 0 111.414 1.414L9.414 8l6.293 6.293a1 1 0 01-1.414 1.414L8 9.414l-6.293 6.293a1 1 0 01-1.414-1.414L6.586 8 .293 1.707a1 1 0 010-1.414z'/%3e%3c/svg%3e")
+    center/1em auto no-repeat;
```
Comment (pradipgharge) [inline css/main.css:130]: I am not sure what this property is

File: css/main.css [hunk 3]
```diff
@@ -117,3 +118,41 @@ button:hover {
...
+  box-shadow: 0 0 0 0.25rem rgba(var(--primary-color), 0.25);
+  opacity: 1;
+}
+
+.ms-2 {
+  margin-left: 0.5rem !important;
+}
+
+.push-to-right {
+  margin-right: auto !important;
```
Comment (05him) [inline css/main.css:153]: is it good to use soo many important?  
or is it kinda compulsory for utility classes?

File: pages/components/Toast/toast.css [hunk 4]
```diff
@@ -0,0 +1,91 @@
+:root {
+  --muted-text: #777;
+  --background-white: rgba(255, 255, 255, 0.85);
+  --border-color: rgba(255, 255, 255, 0.85);
+  --box-shadow: rgba(0, 0, 0, 0.15);
+  --border-bottom: rgba(0, 0, 0, 0.05);
+}
+
+.toast {
+  width: 350px;
```
Comment (pradipgharge) [inline pages/components/Toast/toast.css:10]: use rems for length units

File: css/comp-page.css (hunk 1 above)
Comment (zoef-yt) [inline css/comp-page.css]: var can be used for colours.

File: pages/components/Toast/toast.css [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+.toast-container > :not(:last-child) {
+  margin-bottom: 0.75rem;
+}
+
+.toast-header {
+  display: flex;
+  -ms-flex-align: center;
+  align-items: center;
+  padding: 0.5rem 0.75rem;
+  color: #777;
```
Comment (05him) [inline pages/components/Toast/toast.css]: you should use color variable here to

File: css/comp-page.css (hunk 1 above)
Comment (pradipgharge) [inline css/comp-page.css]: use color variables

**Review by pradipgharge — COMMENTED** [review]
left few comments do check !

File: pages/components/Toast/toast.css (hunk 4 above)
Comment (zoef-yt) [inline pages/components/Toast/toast.css:10]: Use relative units. Such as rem or %.

File: pages/components/Toast/toast.css (hunk 5 above)
Comment (zoef-yt) [inline pages/components/Toast/toast.css]: Use var for colour.

**Review by zoef-yt — CHANGES_REQUESTED** [review]

**Review by 05him — APPROVED** [review]
Your UI is superb , loved the logo on home screen and code is clean 
just few suggestion which I've mentioned 

and one more thing I would like to mention that 
image
try changing the toast heading background color or the color of "5 min ago" as here these both are getting mixed up and a bit hard to read "5 min ago" thing

File: pages/components/Toast/toast.css (hunk 4 above)
Comment (05him) [reply pages/components/Toast/toast.css:10]: she have used max-width:100% too  i think that's why she have used px here

**Comment by SalonSikdar** [comment]
> left few comments do check !

ack, cheking

File: css/main.css (hunk 3 above)
Comment (SalonSikdar) [reply css/main.css:153]: for the global class, my script would work for my demo website without the !important tags.  But in a real-world scenario, should a dev use my CL, these are common styles that may get overridden by the dev's local style sheet/ browser stylesheet. Hence added important to these common styles.

File: pages/components/Toast/toast.css (hunk 5 above)
Comment (SalonSikdar) [reply pages/components/Toast/toast.css]: ack done, missed this one

File: css/comp-page.css (hunk 1 above)
Comment (SalonSikdar) [reply css/comp-page.css]: ack, done

File: pages/components/Toast/toast.css (hunk 5 above)
Comment (SalonSikdar) [reply pages/components/Toast/toast.css]: ack, done

File: pages/components/Toast/toast.css (hunk 4 above)
Comment (SalonSikdar) [reply pages/components/Toast/toast.css:10]: yes, @05him is correct. That's the intention.  we do not need redundant relative sizing

File: css/comp-page.css (hunk 1 above)
Comment (SalonSikdar) [reply css/comp-page.css]: ack, done

Comment (SalonSikdar) [reply css/comp-page.css]: ack, done

File: pages/components/Toast/toast.css (hunk 4 above)
Comment (SalonSikdar) [reply pages/components/Toast/toast.css:10]: i've used a dynamic sizing in percentage.

File: css/main.css (hunk 2 above)
Comment (SalonSikdar) [reply css/main.css:130]: it is the close button cross or X mark SVG import

**Review by pradipgharge — APPROVED** [review]

**Pull request merged**